src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+}
+
+interface Props {
+  todo: Todo,
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id="1"
+      className={classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': todo.completed },
+      )}
+    >
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+
+      <UserInfo todo={todo} key={todo.id} />
```

Comment:
you can add to each todo user (like on [React static list of TODOs](https://github.com/mate-academy/react_static-list-of-todos) task)